controller/array_action/array_mediator_svc.py

```diff
@@ -367,3 +367,19 @@ def unmap_volume(self, volume_id, host_name):
         except Exception as ex:
             logger.exception(ex)
             raise ex
+
+
+    def get_array_iscsi_name(self):
```

Comment:
seems that get_array_iscsi_name for XIV returns a list, see https://github.com/IBM/ibm-block-csi-driver/blob/develop/controller/array_action/array_mediator_xiv.py